spec/requests/donations_requests_spec.rb

```diff
@@ -265,6 +274,17 @@
           expect(response.body).to include("you’ll need to make an adjustment to the inventory as well.")
         end
       end
+
+      context "when editing a donation with an inactive donation site" do
+        let!(:active_donation_site) { create(:donation_site, organization: organization, name: "Active Donation Site") }
+        let!(:inactive_donation_site) { create(:donation_site, organization: organization, active: false, name: "Inactive Donation Site") }
+        let!(:donation_with_inactive_site) { create(:donation, organization: organization, donation_site: inactive_donation_site) }
+
+        it "includes the inactive donation site in the dropdown" do
+          get edit_donation_path(donation_with_inactive_site)
+          expect(response.body).to include(inactive_donation_site.name)
```

Comment:
This no longer checks the alphabetization?

Reply:
I've added a test for this ✔️